AccountSettings/src/locales/zh-CN.ts

```diff
@@ -21,6 +21,7 @@ export default {
   'BLOCK_NAME.basic.phone': '联系电话',
   'BLOCK_NAME.basic.phone-message': '请输入您的联系电话!',
   'BLOCK_NAME.basic.update': '更新基本信息',
+  'BLOCK_NAME.basic.updatesucc': '更新基本信息成功',
```

Comment:
`BLOCK_NAME.basic.update.success`

Reply:
```suggestion
  'BLOCK_NAME.basic.updatesucc': '更新基本信息成功',
```

fix: change BLOCK_NAME.basic.updatesucc to BLOCK_NAME.basic.update.success